docs/whichkey/reference.md

```diff
@@ -0,0 +1,281 @@
+---
+id: reference
+title: Reference
+---
+Which-key extension aims to provides similar keybinding popup using the [QuickPick](https://code.visualstudio.com/api/references/vscode-api#QuickPick)
+API from vscode.
+This extension was broken out of `VSpaceCode`, which was created to emulate the keybinding of spacemacs in VSCode ([VSpaceCode](./../installation.md)),
+to decouple the core and the definition of bindings.
+
+## Commands
+
+Commands are the main way to inference with which-key.
+
+### Show
+
+Command: `whichkey.show`
+
+Args: `string`, `Array<BindingItem>`, `undefined`
+
+Description:
+A command to show which-key menu.
+
+- `string` is the key of the [registered](#register) bindings that will be use to display a which-key menu.
+- `Array<BindingItem>` is an array of [`BindingItem`](#bindingitem) that will be used directly to display a which-key menu.
+- `undefined` means the default bindings defined at `whichkey.bindings` in the configuration will be used to display a which-key menu.
+
+### Register
+
+Command: `whichkey.register`
+
+Args:
+
+```ts
+{
+    bindings: string;
+    overrides?: string;
+    title?: string;
+}
+```
```

Comment:
shouldn't the format of the description of the args be coherent with the other commands?
In the other commands you are not using code to describe them.

Reply:
I thought this would be a little bit easier to read since the description is right below, and I also have a little bit of code in args of `whichkey.triggerKey` in line 52.